catalog-entities/marketplace/plugins/datadog.yaml

```diff
@@ -25,7 +25,9 @@ spec:
   authors:
     - name: Roadie
       url: https://roadie.io/
-  support: tech-preview
+  support:
+    name: Roadie
```

Comment:
If the repo for the plugin is in the backstage community plugins, I think we should set this to "Backstage Community"